docs/get-started/setup.md

```diff
@@ -1,3 +1,109 @@
 ---
 title: 'Setup'
 ---
+
+Now that you’ve learned about what stories are and how to browse stories, let’s demo working on one of your components. Pick a simple component from your project, like a Button, and write a `.stories.js` file to go along with it. It might look something like this:
+
+```js
+// YourComponent.stories.js
+
+import { YourComponent } from ‘./YourComponent’;
+
+// This default export determines where you story goes in the story list
+export default {
+  component: YourComponent,
+  title: ‘YourComponent’,
+}
+
+const Story = (args) => <YourComponent {...args} />;
+
+export FirstStory = Story.bind({})
+FirstStory.args = {
+  /* the args you need here will depend on your component */
+}
+```
+
+Go to your Storybook to view the rendered component. It’s OK if it looks a bit unusual right now.
+
+Depending on your technology stack, you also might need to configure the Storybook environment further. 
+
+
+### Configure Storybook for your stack
+
+Storybook comes with a permissive [default configuration](../configure/overview). It attempts to customize itself to fit your setup. But it’s not foolproof. 
+
+Your project may have additional requirements before components can be rendered in isolation. This warrants customizing configuration further. There are three broad categories of configuration you might need. 
+
+
+#### Build configuration like webpack and Babel
+
+<div style="background-color:#F8FAFC">
+TODO: ask dom/tom presets item points to the preset gallery???
```

Comment:
Hmm, I'm not sure. Are we folding the presets gallery into the addons gallery in the frontpage (https://storybook.js.org/addons/)? 

I'm not sure from a user's perspective there needs to be a big distinction between presets + addons. WDYT @jonniebigodes?

Reply:
I'll run by @domyen and see if we can get a consensus in this.

Reply:
Link to API/Addons#addon-presets

Reply:
updated the link appropriately